### PR TITLE
fix(Image): can not cancel previewMask

### LINKED ISSUE #5524: Image component, close the mask and its content when hovering
- [ ] I have searched the [issues](https://github.com/vueComponent/ant-design-vue/issues)         of this repository and believe that this is not a duplicate.
        
        
### Version
3.2.0

### Environment
vue3

### Reproduction link
[https://www.antdv.com/components/image-cn](https://www.antdv.com/components/image-cn)

### Steps to reproduce
最新版本，hover的时候会出现一个mask，但是之前版本没有
通过现有 api 似乎无法移除mask

### What is expected?
移除 mask

### What is actually happening?
mask 无法移除

### PR DESCRIPTION
**Pull request by buqiyuan**
First of all, thank you for your contribution! 😄

fix #5524

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

**Comment by tangjinzhou** [comment]
忘了说我的想法了：
我们不用支持吧，给他个方案就可以了  不是有 maskClassName 这个了吗，可以满足他了

**Comment by buqiyuan** [comment]
> 忘了说我的想法了： 我们不用支持吧，给他个方案就可以了 不是有 maskClassName 这个了吗，可以满足他了

😂自定义`maskClassName`然后再设置`display: none;`吗？但这样感觉可能就没有将`previewMask`设置为`false`那么符合用户的直观，并且我也体验了一下`vc-image`，
image
在上面这个逻辑中，previewMask的值为`falsy`的时候都不会渲染`mask`，由于我们的`previewMask`是`slot`所以始终为`truthy`。所以我觉得这样也符合`vc-image`原本的一个逻辑，`previewMask`也相对于`maskClassName`更语义化一些，这次提交 我也移除了很多 多余的代码

File: components/image/index.en-US.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Previewable image.
 | placeholder | Load placeholder, use default placeholder when set `true` | boolean \| slot | - | 2.0.0 |
 | preview | preview config, disabled when `false` | boolean \| [previewType](#previewType) | true | 2.0.0 |
 | src | Image path | string | - | 2.0.0 |
-| previewMask | custom mask | slot | - | 3.2.0 |
+| previewMask | custom mask | boolean \| VNode \| VNode[] \| slot | true | 3.2.0 |
```
Comment (tangjinzhou) [inline components/image/index.en-US.md:25]: `boolean  \| function \| slot`   默认值 undefined
只要 previewMask 是 undefined 就取 slot

File: components/image/index.tsx [hunk 2]
```diff
@@ -49,13 +49,17 @@ const Image = defineComponent<ImageProps>({
...
             ...slots,
             previewMask:
-              slots.previewMask ??
-              (() => (
-                <div class={`${prefixCls.value}-mask-info`}>
-                  <EyeOutlined />
-                  {imageLocale?.preview}
-                </div>
-              )),
+              props.previewMask === false
```
Comment (tangjinzhou) [inline components/image/index.tsx:52]: `props.previewMask === false ? null : (props.previewMask ?? slots.previewMask)`

**Review by tangjinzhou — CHANGES_REQUESTED** [review]

File: components/image/index.en-US.md (hunk 1 above)
Comment (buqiyuan) [reply components/image/index.en-US.md]: done

**Pull request merged**